src/component/ColorCard/index.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import styled from "styled-components";
+import TextBox from "../Text";
+const CardWrapper = styled.div`
+  margin: 10px;
+  width: 300px;
+  height: 430px;
+  border: 1px black solid;
+  border-radius: ${(props) =>
+    props.border_radius ? props.border_radius : "10%"}; //
+`;
+
+const ColorWrapper = styled.div`
+  height: 350px;
+  border: 1px solid black;
+  background-color: ${(props) => (props.color ? props.color : "white")};
+  border-top-left-radius: ${(props) =>
+    props.border_radius ? props.border_radius : "10%"};
+  border-top-right-radius: ${(props) =>
+    props.border_radius ? props.border_radius : "10%"};
+  //
+`;
+
+const ColorCard = (props) => {
+  return (
+    <CardWrapper border_radius="40px">
+      <ColorWrapper color={props.color} border_radius="10px" />
+      <TextBox color={props.color} fontsize="50px" textalign="center">
+        {props.color}
```

Comment:
자바스크립트에서는 변수나 함수의 네이밍은 camelCase 를 사용합니다. 이는 props 에서도 마찬가지인데요!

```suggestion
const ColorCard = (props) => {
  return (
    <CardWrapper border_radius="40px">
      <ColorWrapper color={props.color} borderRadius="10px" />
      <TextBox color={props.color} fontsize="50px" textalign="center">
        {props.color}
```

위와 같이 camelCase 를 사용하면 더 좋을 것 같아요 👍👍👍